source/includes/k8s/steps-configure-keycloak-identity-management.rst

```diff
@@ -0,0 +1,283 @@
+- Set :guilabel:`Claim JSON Type` to ``String``
+- Set :guilabel:`Multivalued` to ``On`` - this allows users to inherit any ``policy`` set in their Groups
+- Set :guilabel:`Aggregate attribute values` to ``On`` - this allows users to inherit any ``policy`` set in their Groups
+
+Once created, assign the Client Scope to the MinIO client.
+Navigate to :guilabel:`Clients` and select the MinIO client.
+
+- Select :guilabel:`Client scopes`, then select :guilabel:`Add client scope`.
+- Select the previously created scope and set the :guilabel:`Assigned type` to ``default``.
+
+3) Apply the Necessary Attribute to Keycloak Users/Groups
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You must assign an attribute named ``policy`` to the Keycloak Users or Groups. 
+Set the value to to any :ref:`policy <minio-policy>` on the MinIO deployment.
+
+For Users, navigate to :guilabel:`Users` and select or create the User:
+
+- From :guilabel:`Credentials`, set the user password to a permanent value if not already set
+- From :guilabel:`Attributes`, create a new attribute with key ``policy`` and value of any :ref:`policy <minio-policy>` (``consoleAdmin``)
+
+For Groups, navigate to :guilabel:`Groups` and select or create the Group:
+
+- From :guilabel:`Attributes`, create a new attribute with key ``policy`` and value of any :ref:`policy <minio-policy>` (``consoleAdmin``)
+
+You can assign users to groups such that they inherit the specified ``policy`` attribute.
+If you set the Mapper settings to enable :guilabel:`Aggregate attribute values`, Keycloak includes the aggregated array of policies as part of the authenticated user's JWT token.
+MinIO can use this list of policies when authorizing the user.
+
+You can test the configured policies of a user by using the Keycloak API:
+
+.. code-block:: shell
+   :class: copyable
+
+   curl -d "client_id=minio" \
+        -d "client_secret=secretvalue" \
+        -d "grant_type=password" \
+        -d "username=minio-user-1" \
+        -d "password=minio-user-1-password" \
+        http://keycloak-url:port/realms/REALM/protocol/openid-connect/token
+
+If successful, the ``access_token`` contains the JWT necessary to use the MinIO :ref:`minio-sts-assumerolewithwebidentity` STS API and generate S3 credentials.
+
+You can use a JWT decoder to review the payload an ensure it contains the ``policy`` key with one or more MinIO policies listed.
+
+4) Configure MinIO for Keycloak Authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MinIO supports multiple methods for configuring Keycloak authentication:
+
+- Using the MinIO Operator Console
+- Using the MinIO Tenant Console
+- Using a terminal/shell and the :mc:`mc admin idp openid` command
+
+.. tab-set::
+
+   .. tab-item:: MinIO Operator Console
+
+      You can use the MinIO Operator Console to configure Keycloak as the External Identity Provider for the MinIO Tenant:
+
+      .. include:: /includes/common/common-k8s-connect-operator-console.rst
```

Comment:
I'm also not a fan of how you have tabs nested under the MinIO Operator Console tab. It's sort of visually confusing. Not sure what to do here, since it's an include.
My first thought was to rework this section as dropdowns instead of tabs, but not sure that impacts all the other places you use this content.

![image](https://user-images.githubusercontent.com/53910321/222577141-2825662f-4ec9-496e-99fe-a81bd1c673ca.png)

Reply:
Redone with dropdowns.